packages/teraslice/test/lib/cluster/services/cluster/backends/kubernetes/net-spec.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const net = require('net');
+const Porty = require('porty');
+
+const {
+    waitForTcpPortOpen
+} = require('../../../../../../../lib/cluster/services/cluster/backends/kubernetes/net');
+
+// Run tests with DEBUG=True to see output
+function l(msg) {
+    if (process.env.DEBUG) {
+        // eslint-disable-next-line no-console
+        console.log(msg);
+    }
+}
+
+const logger = {
```

Comment:
Use the `debugLogger` from `@terascope/utils`

```suggestion
const logger = debugLogger('port_utils');
```